pkg/admin/graphql/query.go

```diff
@@ -313,5 +314,33 @@ var query = graphql.NewObject(graphql.ObjectConfig{
 				return graphqlutil.NewConnectionFromResult(lazyItems, result)
 			},
 		},
+		"getUsersByStandardAttribute": &graphql.Field{
+			Description: "Get users by standardAttribute, attributeName must be email, phone_number or preferred_username.",
+			Type:        graphql.NewList(nodeUser),
+			Args: relay.NewConnectionArgs(graphql.FieldConfigArgument{
```

Comment:
1. Do not use relay.NewConnectionArgs. That will result in adding some extra arguments we do not need. You can see those extra arguments schema.graphql.

Reply:
fixed query args type